common/i18n/i18nFactory.js

```diff
@@ -0,0 +1,33 @@
+const i18next = require('i18next')
+
+const enTranslation = require('./resources/en')
+
+const createParams = lang => ({
+  fallbackLng: 'en',
+  debug: true,
```

Comment:
Why debug true?

Reply:
Just for the time being, for development... but right should be removed on master

Reply:
you could use https://github.com/openforis/arena/blob/481d40c0da07e4dc1327761f859cb6398562c318/common/processUtils.js#L5

Reply:
ok, done